front-end/src/App.tsx

```diff
@@ -1,8 +1,9 @@
 import React from 'react';
 import './App.css';
+import { EmployeeRegistrationForm } from './Employee/Registration/employee-registration.component';
 
 function App() {
-  return <div></div>;
+  return <>{<EmployeeRegistrationForm />}</>;
```

Comment:
nit
```suggestion
  return <EmployeeRegistrationForm />
```

No need to wrap the component